spec/requests/api/v0/presentations_request_spec.rb

```diff
@@ -53,18 +54,44 @@
     end
   end
 
+  context "presentation after rating" do
+    # Mix of ratings from this app, another app, target content, other content, target user, other user
+    let!(:rating_1) { create_rating(app: an_app, content_uid: "a", user_uid: "1") }
+    let!(:rating_2) { create_rating(app: an_app, content_uid: "a", user_uid: "1") }
+    let!(:rating_3) { create_rating(app: an_app, content_uid: "b", user_uid: "1") }
+    let!(:rating_4) { create_rating(app: an_app, content_uid: "a", user_uid: "2") }
+    let!(:rating_4) { create_rating(app: another_app, content_uid: "a", user_uid: "1") }
+
```

Comment:
why's `rating_4` listed twice here?  The second instance will overwrite the first I think.

Reply:
oops :-)